splunk/wazuh_cf_splunk.sh

```diff
@@ -11,8 +11,8 @@ eth0_ip=$(/sbin/ifconfig eth0 | grep 'inet addr:' | cut -d: -f2  | cut -d' ' -f1
 wazuh_api_user=$(cat /tmp/wazuh_cf_settings | grep '^WazuhApiAdminUsername:' | cut -d' ' -f2)
 wazuh_api_password=$(cat /tmp/wazuh_cf_settings | grep '^WazuhApiAdminPassword:' | cut -d' ' -f2)
 wazuh_api_port=$(cat /tmp/wazuh_cf_settings | grep '^WazuhApiPort:' | cut -d' ' -f2)
-TAG="v3.11.4"
-APP_TAG="v3.11.4-7.3.4"
+TAG="v3.12.0"
+APP_TAG="v3.12.0-7.3.4"
```

Comment:
The tag should be `v3.12.0-7.6.1`

Reply:
Splunk follows different tagging versions: https://github.com/wazuh/wazuh-splunk/tree/v3.12.0-7.3.4

Reply:
My bad.